### PR TITLE
Fix some locale issues

### PR DESCRIPTION
**Pull request by rainbowdashlabs**
**Checklist**
- [ ] I made changes to commands
  - [ ] I fully localised the command
  - [ ] I fully checked the commands functionality

- [ ] I added new localisation codes
  - [ ] I fully translated it for all languages
  - [ ] I sorted properties alphabetically

- [ ] I made changes to the database
  - [ ] I added my changed to a patch file
  - [ ] I made sure my database was up to date
  - [ ] I checked that the migration works

- [ ] I made changes to internal structure 

**Short Description**
Fixed up some locale codes.  Apparently a property key is not allowed to start with a number as it seems. There were also some odd phrases in the german locale (Still are tho)

### CODE REVIEW

File: src/main/resources/locale_de.properties [hunk 1]
```diff
@@ -132,7 +132,7 @@ command.log.message.message.log=Erhaltene Reputation der Nachricht %ID%
 command.log.message.options.messageid.description=Die ID der Nachricht.
 command.log.received.description=Sehe die erhaltene Reputation eines Nutzers.
 command.log.received.message.log=Erhaltene Reputation von %USER%
-command.log.received.options.user.description=Der Benutzer, der den empfangenen Reputation erhalten soll.
+command.log.received.options.user.description=Der Benutzer, für den die Reputation angezeicht werden soll.
```
Comment (Taucher2003) [inline src/main/resources/locale_de.properties:135]:
```suggestion
command.log.received.options.user.description=Der Benutzer, für den die Reputation angezeigt werden soll.
```

File: src/main/resources/locale_de.properties [hunk 2]
```diff
@@ -6,31 +6,31 @@ command.abuseprotection.context.donor.options.state.description=Setzt du den Wer
 command.abuseprotection.context.receiver.description=Legt fest, ob der Empfänger aktuelle Nachrichten benötigt oder nicht.
 command.abuseprotection.context.receiver.message.false=Der Empfänger benötigt keine aktuellen Nachrichten im Kanal.
 command.abuseprotection.context.receiver.message.true=Der Empfänger muss aktuelle Nachrichten im Kanal haben.
-command.abuseprotection.context.receiver.options.state.description=Setzt du auf "true", werden die Empfängerkontextprüfungen aktiviert.
-command.abuseprotection.description=Zeigt die aktuellen Missbrauchsschutzeinstellungen an.
+command.abuseprotection.context.receiver.options.state.description=Setzt du auf "true", werden die Empfängerkontext überprüft.
```
Comment (Taucher2003) [inline src/main/resources/locale_de.properties:9]:
```suggestion
command.abuseprotection.context.receiver.options.state.description=Setzt du den Wert "true", werden die Empfängerkontext überprüft.
```

File: src/main/resources/locale_de.properties [hunk 3]
```diff
@@ -45,26 +45,26 @@ command.channel.add.description=Fügt einen Reputationskanal hinzu.
...
 command.channel.announcement.info.description=Die aktuellen Einstellungen abrufen
-command.channel.announcement.location.message.otherchannel=Eine Benachrichtigung wird an den konfigurierten Channel gesendet.
+command.channel.announcement.location.message.otherchannel=Eine Benachrichtigung wird in dem konfigurierten Kanal gesendet.
 command.channel.announcement.location.message.samechannel=Eine Benachrichtigung wird in dem Kanal gesendet, in dem die Reputation empfangen wurde.
-command.channel.announcement.state.description=Aktivieren von Rangaufstiegsmeldungen.
+command.channel.announcement.state.description=Aktivieren von Aufstiegsmeldungen.
 command.channel.announcement.state.message.active=Die Benachrichtigung wird gesendet, wenn ein Nutzer aufsteigt.
 command.channel.announcement.state.message.inactive=Es wird keine Benachrichtigung verschickt, wenn ein Nutzer aufsteigt.
-command.channel.announcement.state.options.active.description=Setze true, um Rangaufstiegsmeldungen zu aktivieren.
+command.channel.announcement.state.options.active.description=Setze true, um Aufstiegsmeldungen zu aktivieren.
```
Comment (Taucher2003) [inline src/main/resources/locale_de.properties:57]:
```suggestion
command.channel.announcement.state.options.active.description=Setze "true", um Aufstiegsmeldungen zu aktivieren.
```

File: src/main/resources/locale_de.properties [hunk 4]
```diff
@@ -210,32 +210,32 @@ command.repsettings.emojidebug.message.receiverLimit=Das Empfängerlimit wurde e
...
-command.repsettings.info.message.option.byembed.description=Einbettung der Anfrage zulassen oder verweigern
+command.repsettings.info.message.option.byembed.description=Einbettung zur Reputationvergabe
 command.repsettings.info.message.option.byembed.name=Reputation durch Bestätigungsnachricht
-command.repsettings.info.message.option.byfuzzy.description=Fuzzy-Matching für Reputation zulassen oder verweigern
-command.repsettings.info.message.option.byfuzzy.name=Reputation per Fuzzy-Suche
-command.repsettings.info.message.option.bymention.description=Erwähnungen für die Reputation zulassen oder verweigern
-command.repsettings.info.message.option.bymention.name=Reputation per Erwähnung
-command.repsettings.info.message.option.byreaction.description=Reaktionen für Reputation zulassen oder verweigern
-command.repsettings.info.message.option.byreaction.name=Reputation per Reaktion
+command.repsettings.info.message.option.byfuzzy.description=Unscharfer Vergleich zur Reputationvergabe
```
Comment (Taucher2003) [inline src/main/resources/locale_de.properties:227]: "Unscharfer Vergleich" sounds weird

File: src/main/resources/locale_de.properties [hunk 5]
```diff
@@ -252,30 +252,30 @@ command.roles.add.options.reputation.description=Die Menge an Reputation, die f
 command.roles.add.options.role.description=Die für die Reputation verwendete Rolle.
 command.roles.add.title.added=Rolle hinzugefügt.
 command.roles.description=Verwaltet die Reputationsrollen.
-command.roles.donor.add.description=Fügt eine Spender-Rolle hinzu, welche Reputation erhalten kann.
-command.roles.donor.add.message.add=Spender-Rolle %ROLE% hinzugefügt.
+command.roles.donor.add.description=Fügt eine Spenderrolle hinzu, welche Reputation erhalten kann.
```
Comment (Taucher2003) [inline src/main/resources/locale_de.properties:255]: I think this was wrong?
```suggestion
command.roles.donor.add.description=Fügt eine Spenderrolle hinzu, welche Reputation vergeben kann.
```

File: src/main/resources/locale_de.properties (hunk 4 above)
Comment (rainbowdashlabs) [reply src/main/resources/locale_de.properties:227]: How would you translate "Fuzzy Match"?

Comment (Taucher2003) [reply src/main/resources/locale_de.properties:227]: I don't know, but as it compares by the names, maybe "Namensbasierter Vergleich"?

Comment (yannicklamprecht) [reply src/main/resources/locale_de.properties:227]: Unscharfe Übereinstimmung.

**Review by Taucher2003 — APPROVED** [review]

**Pull request merged**